po/po4a.conf

```diff
@@ -0,0 +1,12 @@
+[po4a_langs] es
+[po4a_paths] po/reuse-docs.pot $lang:po/reuse-docs.$lang.po
+
+[options] opt:"--addendum-charset=UTF-8" opt:"--localized-charset=UTF-8" opt:"--master-charset=UTF-8" opt:"--master-language=en_US" opt:"--msgmerge-opt='--no-wrap'" opt:"--porefs=file" opt:"--wrap-po=newlines"
+
+[po4a_alias:markdown] text opt:"--option markdown" opt:"--option yfm_keys=title" opt:"--addendum-charset=UTF-8" opt:"--localized-charset=UTF-8" opt:"--master-charset=UTF-8" opt:"--keep=100"
+
+[type: markdown] CHANGELOG.md $lang:$lang/CHANGELOG.md
```

Comment:
I don't think we need CHANGELOG.md and README.md to be translated, isn't it?

Reply:
ok makes sense